screens/ItemDetailsScreen.js

```diff
@@ -1,8 +1,8 @@
-import * as WebBrowser from "expo-web-browser";
-import * as React from "react";
-import { StyleSheet, Text, View, TouchableOpacity } from "react-native";
-import { ScrollView } from "react-native-gesture-handler";
-import Colors from "../constants/Colors";
+import * as WebBrowser from 'expo-web-browser';
+import * as React from 'react';
+import { StyleSheet, Text, View, TouchableOpacity } from 'react-native';
+import { ScrollView } from 'react-native-gesture-handler';
+import Colors from '../constants/Colors';
```

Comment:
my prettier changes these back to double quotes 🧐

Reply:
I think mine is set to double quotes, I can switch it to singles in the settings so we don't just go back and forth haha

Reply:
sorry, i mean mine is set to change these all back to doubles, but it looks like this pr changes them all back to singles (so i think your prettier is set to singles?)